--- a/m4/ax_cc_maxopt.m4	2016-07-30 21:21:09.000000000 +0200
+++ b/m4/ax_cc_maxopt.m4	2017-06-20 01:31:53.721258064 +0200
@@ -63,7 +63,6 @@
     
     clang)
         CFLAGS="-O3 -fomit-frame-pointer"
-        AX_CHECK_COMPILER_FLAGS(-mtune=native, CFLAGS="$CFLAGS -mtune=native")
         AX_CHECK_COMPILER_FLAGS(-fstrict-aliasing,CFLAGS="$CFLAGS -fstrict-aliasing")
         AX_CHECK_COMPILER_FLAGS(-ffast-math, CFLAGS="$CFLAGS -ffast-math")
         ;;
@@ -73,9 +72,6 @@
      # Somehow -O3 does not imply -fomit-frame-pointer on ia32
      CFLAGS="-O3 -fomit-frame-pointer"
 
-     # tune for the host by default
-     AX_CHECK_COMPILER_FLAGS(-mtune=native, CFLAGS="$CFLAGS -mtune=native")
-
      # -malign-double for x86 systems
      AX_CHECK_COMPILER_FLAGS(-malign-double, CFLAGS="$CFLAGS -malign-double")

